src/module/actor/actor-sheet.js

```diff
@@ -270,121 +266,71 @@ export class OseActorSheet extends ActorSheet {
     }
-    
   }
   async _onDropItemCreate(itemData, container = false) {
-    const v10 = isNewerVersion(game.version, "10.264");
     //override to fix hidden items because their original containers don't exist on this actor
     itemData = itemData instanceof Array ? itemData : [itemData];
     itemData.forEach((item) => {
-
-      if(v10){
-        if (item.system.containerId && item.system.containerId !== "")
-          item.system.containerId = "";}
-        if (item.type === "container" && typeof item.system.itemIds === "string") {
-          //itemIds was double stringified to fix strange behavior with stringify blanking our Arrays
-          const containedItems = JSON.parse(item.system.itemIds);
-          containedItems.forEach((containedItem) => {
-            containedItem.system.containerId = "";
-          });
-          itemData.push(...containedItems);
-        }
-      if(!v10){
-        if (item.data.containerId && item.data.containerId !== "")
-        item.data.containerId = "";
-        if (item.type === "container" && typeof item.data.data.itemIds === "string") {
-          //itemIds was double stringified to fix strange behavior with stringify blanking our Arrays
-          const containedItems = JSON.parse(item.data.data.itemIds);
-          containedItems.forEach((containedItem) => {
-            containedItem.data.data.containerId = "";
-          });
-          itemData.push(...containedItems);
-        }
-      };
-
-      
-    })
+      if (item.system.containerId && item.system.containerId !== "")
+        item.system.containerId = "";
+      if (
+        item.type === "container" &&
+        typeof item.system.itemIds === "string"
+      ) {
+        //itemIds was double stringified to fix strange behavior with stringify blanking our Arrays
+        const containedItems = JSON.parse(item.system.itemIds);
+        containedItems.forEach((containedItem) => {
+          containedItem.system.containerId = "";
+        });
+        itemData.push(...containedItems);
+      }
+    });
     if (!container) {
       return this.actor.createEmbeddedDocuments("Item", itemData);
     }
-    if (container){
-      let itemIds = v10 ? container.system.itemIds : container.data.data.itemIds;
-      itemIds.push(itemData.id);
-      const item = this.actor.items.get(itemData[0]._id);
-      if(v10){
-        await item.update({system:{containerId: container.id}});
-        await container.update({system:{itemIds: itemIds}});
-      }
-      if(!v10){
-        await item.update({data:{data:{containerId: container.id}}});
-        await container.update({data:{data:{itemIds: itemIds}}});
-      }
 
-    }
+    let itemIds = container.system.itemIds;
+    itemIds.push(itemData.id);
+    const item = this.actor.items.get(itemData[0]._id);
+    await item.update({ system: { containerId: container.id } });
+    await container.update({ system: { itemIds: itemIds } });
```

Comment:
There are some places where we return promises, and others where we await promises in place at the end of a function. I don't think we have to standardize it for this PR, especially if we're not sure how long this sheet is going to stick around, but it might be good to hammer out the pattern we want to use.